src/components/Pages/SignupForm/index.js

```diff
@@ -26,7 +28,7 @@ export const SignupForm = () => {
       navigate("/");
     } catch (error) {
       console.log("error----->", error);
-      setError(error.message);
+      // setError(error.message);
```

Comment:
Is this to handle errors from Firebase? If so, keep it, since it might be a work in progress